aws_advanced_python_wrapper/utils/rdsutils.py

```diff
@@ -59,134 +61,155 @@ class RdsUtils:
     """
 
     AURORA_DNS_PATTERN = r"(?P<instance>.+)\." \
-                         r"(?P<dns>proxy-|cluster-|cluster-ro-|cluster-custom-)?" \
+                         r"(?P<dns>proxy-|cluster-|cluster-ro-|cluster-custom-|limitless-)?" \
```

Comment:
In Java, all patterns end with "$". That makes sure that domain name are completely match patterns, not just starts with a pattern, or contains a pattern.

Reply:
Just to confirm we should expect $ for this pattern?